vumi_http_retry/workers/api/worker.py

```diff
@@ -42,30 +43,35 @@ def setup(self):
     def teardown(self):
         self.redis.transport.loseConnection()
 
-    @classmethod
-    def respond(cls, req, data, code=http.OK):
-        req.responseHeaders.setRawHeaders(
-            'Content-Type', ['application/json'])
-
-        req.setResponseCode(code)
-        return json.dumps(data)
-
     @app.route('/health', methods=['GET'])
     def route_health(self, req):
-        return self.respond(req, {})
+        return response(req, {})
 
     @app.route('/requests/', methods=['POST'])
+    @json_body
+    @validate(
+        has_header('X-Owner-ID'),
+        body_schema({
+            'properties': {
```

Comment:
> Should we put the schema in a constant and reference it here (that way the schema is separately accessible for testing, etc)?

I kind of like it being right next to the route being described, seems more readable. I also don't see us using it as a constant from the tests.

> Should we have a 'type': 'object' at the top-level

+1

> Should we add a 'items': {'type': 'integer', 'minimum': 0} to our intervals

+1

> Should we add a 'additionalProperties': {'type': 'array', 'items': 'str'} to our headers?

+1